packages/eslint-plugin/src/rules/prefer-string-starts-ends-with.ts

```diff
@@ -0,0 +1,652 @@
+/**
+ * @fileoverview Enforce to use `String#startsWith` and `String#endsWith` over other options.
+ * @author Toru Nagashima <https://github.com/mysticatea>
+ */
+
+import { TSESTree } from '@typescript-eslint/typescript-estree';
+import {
+  isNotClosingParenToken,
+  getPropertyName,
+  getStaticValue,
+} from 'eslint-utils';
+import { RegExpParser, AST as RegExpAST } from 'regexpp';
+import { RuleFixer, RuleFix } from 'ts-eslint';
+import ts from 'typescript';
+import { createRule, getParserServices } from '../util';
+
+const EQ_OPERATORS = /^[=!]=/;
+const regexpp = new RegExpParser();
+
+export default createRule({
+  name: 'prefer-string-starts-ends-with',
+  defaultOptions: [],
+
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'enforce to use `String#startsWith` and `String#endsWith` over other options',
+      category: 'Best Practices',
+      recommended: false,
+    },
+    messages: {
+      preferStartsWith: "Use 'String#startsWith' method instead.",
+      preferEndsWith: "Use 'String#endsWith' method instead.",
```

Comment:
```suggestion
      preferEndsWith: "Use the 'String#endsWith' method instead.",
```

And the same thing for `preferStartsWith`.